modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -353,17 +353,11 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
     sys: InstrumentSystem[IO]
   ): TrySeq[List[System[IO]]] = {
     stepType match {
-      case StepType.CelestialObject(inst) => for {
-        tcs  <- getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
-          TcsController.LightSource.Sky, config)
-        gcal <- Gcal.fromConfig(systems.gcal, site == Site.GS)(config)
-      } yield List(sys, tcs, gcal)
+      case StepType.CelestialObject(inst) => getTcs(inst.hasOI.fold(allButGaos, allButGaosNorOi), useGaos = false, sys,
```

Comment:
There are a bit hard to read, could we perhaps write the `.map` in a different line and the parametrs for `getTcs` each on its own line?

Reply:
Done